handler/api/src/main/java/io/knotx/fragments/handler/api/ActionConfig.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2019 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.knotx.fragments.handler.api;
+
+import io.knotx.fragments.handler.api.actionlog.ActionLogMode;
+import io.vertx.core.json.JsonObject;
+
+public class ActionConfig {
+  private final String alias;
+  private final Action doAction;
+  private final ActionLogMode actionLogMode;
+  private final JsonObject options;
+
+  public ActionConfig(String alias, Action doAction, JsonObject options, ActionLogMode actionLogMode) {
+    this.alias = alias;
```

Comment:
Can we rename `alias` to `namespace`? 
@Skejven what is your opinion?

The `alias` name hides what it really is. In most cases it is action name. However, when we introduce template nodes we would get many instances for the same action with different `namespaces` such as:
```
- get-product-1024
- get-product-2048
- get-product-4096
```
So in fact it is not `action name`, something more like `alias` or maybe `namespace`.

Reply:
We can skip this idea now.

Reply:
will you add separate task for that?